packages/gatsby/src/commands/build.ts

```diff
@@ -14,6 +14,7 @@ import { initTracer, stopTracer } from "../utils/tracer"
 import db from "../db"
 import { store, readState } from "../redux"
 import * as appDataUtil from "../utils/app-data"
+import * as pageDataUtil from "../utils/page-data"
```

Comment:
Rather than importing `*` from this and webpack-status, can you do named imports?

Reply:
We actually did that on purpose because `pageDataUtil.write` reads with more context than simply `write`

But then we renamed `write` to `writePageData` so maybe that doesn't matter anymore?

Reply:
Yeah, I'd alias the import in that sort of case. i.e. `import { write as writePageData } from "../"`

Reply:
That's fair. Let me quickly push a fix.

Reply:
I still `kinda` prefer `webpackStatusUtil.markAsPending()` as opposed to `markWebpackStatusAsPending`

Thoughts?

(Changed for page data)

Reply:
Just to comment on this I don't mind using namespace vs individual named imports - but if we go named imports route - please alias them to be more meaningful (or name exports more meaningfully), because right now we have "random", `flush()` somewhere in `build.ts` which you need to check where it's imported from etc

Reply:
If you've seen my recent PRs you'll know I'm a fan of long, descriptive function names!

Reply:
Haha, I know. Okay, I'm convinced. Renaming these now.